tests/integrations/conftest.py

```diff
@@ -0,0 +1,93 @@
+import logging
+
+import pytest
+import pytest_asyncio
+from starkware.starknet.testing.contract import DeclaredClass, StarknetContract
+from starkware.starknet.testing.starknet import Starknet
+
+from tests.utils.utils import (
+    get_contract,
+    hex_string_to_bytes_array,
+    traceit,
+    wrap_for_kakarot,
+)
+
+logger = logging.getLogger()
+
+
+@pytest_asyncio.fixture(scope="module")
+async def kakarot(
+    starknet: Starknet, eth: StarknetContract, contract_account_class: DeclaredClass
+) -> StarknetContract:
+    return await starknet.deploy(
+        source="./src/kakarot/kakarot.cairo",
+        cairo_path=["src"],
+        disable_hint_validation=False,
+        constructor_calldata=[
+            1,
+            eth.contract_address,
+            contract_account_class.class_hash,
+        ],
+    )
+
+
+@pytest_asyncio.fixture(scope="module", autouse=True)
+async def set_account_registry(
+    kakarot: StarknetContract, account_registry: StarknetContract
+):
+    await account_registry.transfer_ownership(kakarot.contract_address).execute(
+        caller_address=1
```

Comment:
we are not allowed to say `caller_address=kakarot.contract_address`
from the beginning?

Reply:
actually the `account_registry` is owned by `1` while here we need to make it owned by kakarot, so we update it when using kakarot and do a cleaning step afterwards (after the yield is the teardown)

see other usage of the fixture in tests/units/test_account_registry.py